src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -182,6 +182,38 @@ object Scanners {
      */
     var revComments: List[Comment] = Nil
 
+    /** The currently closest docstring, replaced every time a new docstring is
+     *  encountered
+     */
+    var closestDocString: List[List[Comment]] = List(List())
```

Comment:
could you please elaborate on why it this a `List[List[Comment]]`? the type seems no to correspond to name\comment.